jsonschema-validation.xml

```diff
@@ -1043,8 +1043,14 @@
                     is going to be removed in the future.
                 </t>
                 <t>
-                    A root schema containing "deprecated" with a value of true indicates the entire
-                    root schema MAY be removed in the future.
+                    A root schema containing "deprecated" with a value of true indicates that
+                    the entire resource beng described MAY be removed in the future.
+                </t>
+                <t>
+                    When the "deprecated" keyword is used to describe an array, it's meaning will
```

Comment:
/is used to describe an array/is applicable to an array/

I figure we have the language of applicability so we should use it.

Reply:
Have you seen this little fella? It's a really good feature if you've not tried poking it before.

<img width="353" alt="image" src="https://user-images.githubusercontent.com/67381/61544467-d389ce80-aa45-11e9-897d-390ff0319761.png">

Reply:
I knew it was there but I didn't know what it looked like. Ta!

Reply:
I was just pushing this fix haha thanks!